google-map-marker.html

```diff
@@ -366,8 +366,10 @@
         subtree: true
       });
 
-      var content = this.innerHTML.trim();
-      if (content) {
+      // Use this element's content DOM node for the info window content.
```

Comment:
I'd still like Line 41: `<template><div><content></content></div></template>` changed.

The idea is that users write:
```
<google-map-marker>
  <div>...</div> <!-- wrapper -->
</google-map-marker>
```
But you already have a wrapper inside shadow dom. Can we remove the one in shadow dom and update the docs to explain to users that they need to use a single wrapper in their light dom. We'll also need to rev a minor version b/c this is a breaking change.